messages/core/position.json

```
@@ -0,0 +1,15 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "type": "array",
+    "title": "Position",
+    "description": "Position [longitude, latitude, altitude] [degrees, degrees, m] given as longitude and latitude in the WGS84 reference system and optionally altitude given as the height above the WGS84 ellipsoid. If altitude is not given, position should be interpreted as being on the local ground/sea level.",
```

I think its better to always give three values. 

```suggestion
    "description": "Position [longitude, latitude, altitude] [degrees, degrees, m] of a body with respect to the WGS84 frame of reference. Altitude is the height of the body above the WGS84 ellipsoid.
```